src/v/pandaproxy/schema_registry/sharded_store.cc

```diff
@@ -131,25 +132,41 @@ sharded_store::get_versions(const subject& sub, include_deleted inc_del) {
     co_return std::move(versions).value();
 }
 
-ss::future<std::vector<schema_version>>
-sharded_store::delete_subject(const subject& sub, permanent_delete permanent) {
+ss::future<std::vector<schema_version>> sharded_store::delete_subject(
+  seq_marker marker, const subject& sub, permanent_delete permanent) {
     auto versions = co_await _store.invoke_on(
-      shard_for(sub), &store::delete_subject, sub, permanent);
+      shard_for(sub), &store::delete_subject, marker, sub, permanent);
     co_return std::move(versions).value();
 }
 
+ss::future<is_deleted> sharded_store::is_subject_deleted(const subject& sub) {
+    auto deleted = co_await _store.invoke_on(
```

Comment:
Can you ensure all the `invoke_on`s pass the `_smp_opts`.

Reply:
Done.